### ISSUE TITLE
Dog park is too prominent

### ISSUE THREAD

**Issue by polarbearing** [gravitystorm/openstreetmap-carto#3041]
Dog park was added in #2216, with colour changed to playground in #2268, with discussions in #341, #2250 and other places.

Currently is works reasonably with small, single-function parks. 
For larger areas, the pattern symbol 'paw' is very strong, it is more like lots of icons. 

When dog park is an additional function to another area, commonly a forest or a park, it becomes problematic. In the forest case, the patterns interfere badly, and the leisure colour prevails over the main colour of the basic feature:
![dog_over_forest](https://user-images.githubusercontent.com/8266355/35508757-d600df1e-04f1-11e8-9db3-8c456fbd656a.png)

People [started to remove the tags](https://www.openstreetmap.org/changeset/55448617) because the prominence annoyed them.

For the small features, a single icon would be sufficient. For the larger ones they would be candidates for an outline, however a single icon would be lost when zooming in. 

Or maybe a lighter pattern but no fill at all, so the pattern shows on whatever land the feature is on. For the forest case, would it be possible to sync the paw and the tree patterns?

**Comment by dieterdreist** [comment]
I believe tagging significant or all parts of an area like the Grunewald
forest is abusing the tag. Just because dogs can (supposedly) be unleashed,
this isn't a dog park. Compare to the sahara, should we tag this all as dog
park because you can let your unleashed dogs play?

**Comment by kocio-pl** [comment]
Accidentally I was [talking about this specific area](https://www.openstreetmap.org/changeset/40873183) about 2 months ago and for me a [sign in German](https://www.top10berlin.de/sites/top10berlin.de/files/styles/juicebox/public/location/slider/2014/09/08/slider_hundeauslaufgebiet_grunewald_top10_1.jpg?itok=I5i-knkt) was enough to leave a dog park tag.

It seems to me that even big dog park would look OK, the problem is mixing with trees. I don't know how to solve this.

**Comment by meased** [comment]
Agree that the screenshot is not pretty. Is this a dog park problem, though? Or is there a general problem of overlapping pattern symbols? (What about cemetery and trees?)

**Comment by polarbearing** [comment]
Single icon such as playground, and outline instead of fill. There is no reason why kids get just ine icon, and dogs the pattern all over ;-) The dogs are not growing there like in an orchard. 
Icon appearing depending on way_size.

Before:
![dogs_tempelhof](https://user-images.githubusercontent.com/8266355/35575434-aaa5ff16-05dc-11e8-8db2-12a12e99668b.png)

After:
![dogs_tempelhof_outline](https://user-images.githubusercontent.com/8266355/35575476-bb2fcb00-05dc-11e8-8f34-aff741b9a881.png)

![z13_dogs_grunewald](https://user-images.githubusercontent.com/8266355/35575483-c09819ee-05dc-11e8-8bd8-fc4f4fc65a8b.png)

**Comment by kocio-pl** [comment]
Just the icon makes sense for me, but why did you remove the leisure background? The new outline is similar to nature reserves - especially once you zoom out to z15 and lower, where the icon is not rendered:

https://github.com/gravitystorm/openstreetmap-carto/blob/d56f1a7ff4f608c1101218905619e9a1ce3941e1/landcover.mss#L264-L276

**Comment by polarbearing** [comment]
The outline was just a 60-second-draft to show the concept. It would need to be compared with other outlines in #3045. 
The icon can be shown, based on way_size, earlier than z15. In the Grunewald example above it is shown at z13.
The fill was dropped, since the feature can be single-purpose (the small fenced area on Tempelhofer Feld) or secondary purpose (the clearly defined boundaries in Grunewald being primarily a forest). So the natural environment would be visible: sand, grass, forest...

**Comment by kocio-pl** [comment]
I think the outline could be in the current color for other leisure areas. 

I'm not sure if dropping the fill just because it can be second purpose is the proper reason - how do we tag the primary role in general? We don't, we just use two or more relevant tags. Nothing forbids adding such details for filled areas and it's done in reality (sport centers, playgrounds) - it can be even tree area, of course.

I realize that it's the Grunewald look that pushes us to find something suitable. But if we drop the dog park pattern, we would see leisure area with trees (not clashing with other symbols), which is exactly what it is. Are there more such places where it can be not nice visually? If we want to break the system, it would be good to have more reasons than just one case.

**Comment by polarbearing** [comment]
the dog usage is a specific designation of a variety of grounds. I'm still in favour of indicating that usage with the outline, and leave the fill for what covers the ground.

**Comment by kocio-pl** [comment]
OK, we might go this way, since it's a corner case and it's alway possible to add leisure/landuse, but we need a good outline anyway. Could you provide it (and PR if possible)?

**Comment by matkoniecz** [comment]
> I realize that it's the Grunewald look that pushes us to find something suitable. But if we drop the dog park pattern, we would see leisure area with trees (not clashing with other symbols), which is exactly what it is. Are there more such places where it can be not nice visually? If we want to break the system, it would be good to have more reasons than just one case.

Can anybody provide more examples where entire forest was designated as a dog park ("designated area, with or without a fenced boundary, where dog-owners are permitted to exercise their pets unrestrained")?

**Comment by dieterdreist** [comment]
I agree the definition is too inclusive. By that, we could tag the whole of
antarctica as leisure=dog_park

**Comment by kocio-pl** [comment]
I guess it's not designated area - Grunewald has special plates, Antarctica probably doesn't.

**Comment by Tar-Calion** [comment]
Here are other examples where dog park is conflicting with grass and forest: 
[https://www.openstreetmap.org/#map=17/50.98569/6.86244&layers=N](https://www.openstreetmap.org/#map=17/50.98569/6.86244&layers=N)
![1](https://user-images.githubusercontent.com/43185234/45382677-439b1d00-b609-11e8-96f0-784f7146bc88.PNG)

[https://www.openstreetmap.org/#map=19/50.97648/6.86654&layers=N](https://www.openstreetmap.org/#map=19/50.97648/6.86654&layers=N)
![2](https://user-images.githubusercontent.com/43185234/45382782-9543a780-b609-11e8-87b7-749f14ba34f9.PNG)

**Comment by jeisenbe** [comment]
@polarbearing, are you going to continue work on this? It would be great to fix this issue.

**Comment by polarbearing** [comment]
I'm still interested, but short of time currently. Still favouring the outline version without fill.

**Comment by HolgerJeromin** [comment]
> Are there real cases where a dog park is in a forest? 

In the complex real world you always find places which are exceptions :-)
https://walkyourdog.de/hundeauslaufgebiet-grunewald/

But the question is: how often are they to adjust tagging to these rare (?) features.

**Comment by polarbearing** [comment]
Still the pattern instead of a single icon is inconsistent with other POIs, see comparison with playground example above.

**Comment by ZeLonewolf** [comment]
If we would tag that as a dog park in the USA, we would also have to tag vast areas of the country which are wooded areas that permit you to walk your dog :)  I would consider the Hundeauslaufgebiet-Grunewald example to be a mis-tagging that should instead be tagged `dog=yes` as it does not (to me) meet the duck test of a dog park, which are small and enclosed areas -- not just places where dog walking is allowed!  But, that is off-topic here.

I do agree that the current dog park rendering just looks poor on the map and is inconsistent with other features, regardless of my feeling about what should be considered a dog park.  I would rather see the pattern go away and the paw print used as a POI icon.

**Comment by dieterdreist** [comment]
sent from a phone

> On 20 Apr 2021, at 11:34, polarbearing ***@***.***> wrote:
> 
> Still the pattern instead of a single icon is inconsistent with other POIs, see comparison with playground example above.


while I could buy this for the usual dog park around here (some squaremetres of a park, fenced off for dog owners and their pets), it’s less pertinent for a big area like the linked Hundesauslaufgebiet Grunewald, which isn’t exclusively used by dog owners either. A visualization as property of the land makes sense. 
Dog parks have a distinct fill (like playgrounds) but no icon. They are shown like parks (name, fill, casing, no icon).

**Comment by dieterdreist** [comment]
sent from a phone

> On 20 Apr 2021, at 14:49, Brian Sperlongano ***@***.***> wrote:
> 
> I would consider the Hundeauslaufgebiet-Grunewald example to be a mis-tagging that should instead be tagged dog=yes as it does not (to me) meet the duck test of a dog park, which are small and enclosed areas -- not just places where dog walking is allowed!


somehow agree, but the situation is quite exceptional there compared to the rest of the country, where you always have to leash your dog in the forest (unless you’re the forester or hunter I guess). That’s the legal situation (in practice, you might encounter unleashed dogs in the forest also in Germany).

**Comment by ZeLonewolf** [comment]
> Dog parks have a distinct fill (like playgrounds) but no icon. They are shown like parks (name, fill, casing, no icon).

This did not appear to be true from my experimentation today.  Dog parks render an icon (a paw, just like pet shops) at zoom levels 17-19, which seems like reasonable zoom range for typical-sized urban dog parks.  I've posted a PR above which unifies the dog park render style with playgrounds, with the exception of the different icon, and maintains the dog park text label at lower zooms for larger areas.

I could probably use a few additional test examples, particular if there are larger-than-usual standalone dog parks that aren't obscured with all these extra features.

**Comment by UgnilJoZ** [comment]
I hope it is okay to share my personal opinion that rendering a forest as light green is weird, whether it intersects with a dog park or not.

It just happened again that someone [removed](https://osm.org/changeset/120346211) `leisure=dog_park` from an object because the rendering was confusing.

IMHO, the physical feature "forest" should dominate over the legal aspect (the permission to walk dogs without a leash).

**Comment by rohieb** [comment]
I just stumbled on this again around https://www.openstreetmap.org/note/4182326 after I deleted a lot of overlapping landuse=forest. Is there maybe a way to prefer rendering forests over dog parks?
